guide/ops/locations/_GCE.md

```diff
@@ -43,4 +82,10 @@ For example, for dev/demo purposes an "everything" network could be created that
 || Source IP Ranges            || 0.0.0.0/0                   |
 || Allowed protocols and ports || tcp:0-65535 and udp:0-65535 |
 
+To configure the location to use this, you can include a location configuration option like 
+`networkName: everything`.
```

Comment:
Thanks for checking thoroughly/properly! I had an example kicking around in my catalog that said "networkName", but I hadn't tested it recently (ever?!). I'll change this to your example.